factory/processing/processComponents.go

```diff
@@ -2060,6 +2063,9 @@ func (pc *processComponents) Close() error {
 	if !check.IfNil(pc.blockProcessor) {
 		log.LogIfError(pc.blockProcessor.Close())
 	}
+	if !check.IfNil(pc.blocksQueue) {
+		pc.blocksQueue.Close()
```

Comment:
use `log.LogIfError` as for the other fields?

Reply:
blocksQueue.Close() does not return error